Sources/TuistGraph/Models/Project.swift

```diff
@@ -59,6 +43,8 @@ public struct Project: Hashable, Equatable, CustomStringConvertible, CustomDebug
     /// IDE template macros that represent content of IDETemplateMacros.plist
     public var ideTemplateMacros: IDETemplateMacros?
 
+    public let resourceSynthesizers: [ResourceSynthesizer]
```

Comment:
Shouldn't this be also documented? 🤔

Reply:
I'll add it but would not be a huge issue since `TuistGraph` models are used only internally